backend/compact-connect/lambdas/python/staff-users/tests/function/test_handlers/test_patch_user.py

```diff
@@ -105,6 +105,43 @@ def test_patch_user_document_path_overlap(self):
             user,
         )
 
+    def test_patch_user_add_to_empty_actions(self):
+        from handlers.users import patch_user, post_user
+
+        with open('tests/resources/api-event.json') as f:
+            event = json.load(f)
+
+        with open('tests/resources/api/user-post.json') as f:
+            api_user = json.load(f)
+        # Create a user with no compact read or admin, no actions in a jurisdiction
+        api_user['permissions'] = {'aslp': {'jurisdictions': {}}}
```

Comment:
Wouldn't we want to instead (or additionally) test a permission structure like the below (all false)?

Perhaps (1) create user with no permissions, (2) add permissions, (3) patch user with something like the below, expecting a user again with no permissions?

(This more closely mirrors my test case, but you understand the codebase here better so I defer to your expertise.)

```json
{
    "permissions": {
        "aslp": {
            "actions": {
                "read": false,
                "admin": false
            },
            "jurisdictions": {
                "ak": {
                    "actions": {
                        "write": false,
                        "admin": false
                    }
                },
                "id": {
                    "actions": {
                        "write": false,
                        "admin": false
                    }
                },
                "ut": {
                    "actions": {
                        "write": false,
                        "admin": false
                    }
                }
            }
        }
    }
}
```

Reply:
Sure, adding this test case wouldn't hurt. I'll add it in.

Reply:
Sounds good. Once that's done, we can get this merged in, followed by #401